databusclient/api/delete.py

```
@@ -0,0 +1,188 @@
+    - dry_run: If True, do not perform the deletion but only print what would be deleted
+    - force: If True, skip confirmation prompt and proceed with deletion
+    """
+
+    # Confirm the deletion request, skip the request or cancel deletion process
+    if not (dry_run or force):
+        action = _confirm_delete(databusURI)
+        if action == "skip":
+            print(f"Skipping: {databusURI}\n")
+            return
+        if action == "cancel":
+            raise KeyboardInterrupt("Deletion cancelled by user.")
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)